data/builder/file.go

```diff
@@ -0,0 +1,262 @@
+package builder
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/ipfs/go-cid"
+	chunk "github.com/ipfs/go-ipfs-chunker"
+	"github.com/ipfs/go-unixfsnode/data"
+	dagpb "github.com/ipld/go-codec-dagpb"
+	"github.com/ipld/go-ipld-prime"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+	"github.com/multiformats/go-multicodec"
+	multihash "github.com/multiformats/go-multihash/core"
+)
+
+// BuildUnixFSFile creates a dag of ipld Nodes representing file data.
+// This recreates the functionality previously found in
+// github.com/ipfs/go-unixfs/importer/balanced, but tailored to the
```

Comment:
I guece we can live without trickle dag for the moment... I wonder if we even ever use this.